R/check_params.R

```diff
@@ -233,6 +233,11 @@ check_params = function(params_sleep = c(), params_metrics = c(),
       }
     }
   }
+  if (length(params_output) > 0) {
+    if (!("WW" %in% params_output[["timewindow"]])) {
+      warning("\nNote: the part4 report will only include variable nonwear_perc_spt if argument timewindow is set to 'WW' or c('MM', 'WW').", call. = FALSE)
```

Comment:
This means that users who intentionally always only use MM window will always get this warning. I think we should reserve warnings only to warn users about problems with data or mistakes in how GGIR is used and not to explain GGIR behaviour.

We do not promise in the documentation that this variable is stored in the part 4 output. So, if we do not store it the user will not miss it.

How about we instead update the vignette documentation on the part4 output and clarify that if part 5 has been run with timewindow = WW then part 4 will copy nonwear_perc_spt from the part 5 output.

Reply:
Ok, I have reversed this change and updated the vignette. I also fixed #684 in the last commit. Not sure why the tests aren't passing, but I cannot work on this right now because I am processing a bunch of files. Hopefully it is finished by tomorrow and I will then revise this.